### PR TITLE
EN: Training Quest descriptions + misc

### PR DESCRIPTION
**Pull request by DabirSA**

### CODE REVIEW

File: UI/quest_counter.csv [hunk 1]
```diff
@@ -172,16 +172,16 @@ HelpMes19#0,"""A limited-time ARKS GP Quest<br>which can be accepted from a<br>c
...
+HelpMes30#0,"""Use this combat scenario to practice<br>using the \""Bouncer\"" class in battle.<br><br>\""Bouncer\"" is a class that specializes in<br>tricky attacks using Dual Blades, Jet<br>Boots and even Techniques.<br><br>※In Training Quests, your character's<br><yellow>stats and equipment will be set.<c><br>Additionally, you cannot access storage.<br><br>※You may return to a different block."""
+HelpMes31#0,"""Use this combat scenario to practice<br>using the \""Summoner\"" class in battle.<br><br>\""Summoner\"" is a class that focuses on<br>training Pets and using Tacts to have<br>them fight for you.<br><br>※In Training Quests, your character's<br><yellow>stats and equipment will be set.<c><br>Additionally, you cannot access storage.<br><br>※You may return to a different block."""
```
Comment (alama) [inline UI/quest_counter.csv]: takt or Tacts?

File: Quests/Descriptions/quest_00160300.csv [hunk 2]
```diff
@@ -1,14 +1,14 @@
 Title#0,"""Force Basics"""
-Explanation#0,"""■訓練メニュー<br>――――――――――――――――――――<br>課題１：テクニックのジャストアタック<br>課題２：テクニックの連続ジャストアタック<br>課題３：チャージTECのジャストアタック<br>課題４：タリスの使用<br>課題５：実戦訓練"""
+Explanation#0,"""■Training Program<br>――――――――――――――――――――<br>Task 1: Technique Just Attacks<br>Task 2: Consecutive Technique JAs<br>Task 3: Charged Technique JAs<br>Task 4: Using Talises<br>Task 5: Live combat"""
```
Comment (alama) [inline Quests/Descriptions/quest_00160300.csv:2]: JAs?

Comment (DabirSA) [reply Quests/Descriptions/quest_00160300.csv:2]: Just Attacks doesn't quite fit and the first task was already about Just Attacks so I reckon it's fine

File: UI/quest_counter.csv (hunk 1 above)
Comment (DabirSA) [reply UI/quest_counter.csv]: fixed

File: Quests/Descriptions/quest_00160300.csv (hunk 2 above)
Comment (alama) [reply Quests/Descriptions/quest_00160300.csv:2]: ok, adding "JAs" to Misc word list

**Review by SynthSy — APPROVED** [review]

**Pull request merged**